nibabel/tests/test_funcs.py

```diff
@@ -193,3 +193,22 @@ def test_closest_canonical():
     img.header.set_dim_info(None, None, 2)
     xyz_img = as_closest_canonical(img)
     assert_true(xyz_img.header.get_dim_info() == (None, None, 1))
+
+
+def test_crop_image():
```

Comment:
It would be good to test a series of crops / subsamples here.

Am I right that you're not testing the `img.slicer` functionality here?

Reply:
Not yet, no. Wanted to get the API before working on the tests.